src/client/containers/LandingPage/LandingPage.Style.css

```
@@ -1,3 +1,13 @@
-h1 {
-  color: aqua;
+#landing-main {
+  width: 95%;
+  margin: auto;
+  padding: 2em 0;
+}
+
+#landing-main .loading {
+  text-align: center;
+}
+
+#landing-main .footer {
+  padding-top: 2em;
```

I would suggest not styling on IDs. Use IDs to identify unique sections of the page only. You can add classes with the same names as the IDs and style the classes instead. This helps with not driving up the specificity of your CSS and makes it easier if you need to override things in the future.

```suggestion
.landing-main {
  width: 95%;
  margin: auto;
  padding: 2em 0;
}

.landing-main .loading {
  text-align: center;
}

.landing-main .footer {
  padding-top: 2em;
```